CHANGELOG-4.1.md

```diff
@@ -12,11 +12,15 @@
 [#14696](https://github.com/phalcon/cphalcon/issues/14696)
 - Added `Phalcon\Http\Request::getPreferredIsoLocaleVariant()` to return the base language if this is a specific one (`en` vs `en-US`) [#3135](https://github.com/phalcon/cphalcon/issues/3135)
 - Added `preload` for Volt, which will send a HTTP/2 preload header [#13128](https://github.com/phalcon/cphalcon/issues/13128)
+- Added `Phalcon\Mvc\Model::getModelSetup()` to return current model setup
+- Removed `phalcon.orm.ast_cache` because it wasn't used anywhere
```

Comment:
It seems that this is used in `orm.c`  phalcon/mvc/model/orm.c

Reply:
Yea, only in phalcon_orm_destroy_cache, there is nothing to added to it so it will be never triggered. I removed code with ti from there too.

Reply:
Could you move this to `## Removed section`